### PR TITLE
Initial Tests for Advanced Forum. travis support

### PR DESCRIPTION
**Pull request by podarok**
Steps for review
- install site by

``` sh
sh reinstall.sh
```
- install Drupal Console
- go to docroot folder and run 

``` sh
drupal -vvv test:run --url http://drupal8.192.168.56.132.xip.io/ "\Drupal\advanced_forum\Tests\AdvancedForumControllerTest"
```

or use Testing module to run advanced_forum group of tests

for getting green tests result
image

### CODE REVIEW

**Comment by alexschedrov** [comment]
@podarok Will be easier to have own CI :)

**Comment by podarok** [comment]
well. the small bug only... If it fixed - all should work

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+  - travis_retry composer selfupdate
+
+install:
+  # Install Ansible.
+  - sudo pip install ansible
+  - travis_retry composer install --no-interaction
+  - wget http://ua2.php.net/get/php-5.6.16.tar.gz/from/this/mirror -O php-5.6.16.tar.gz
+  - tar -xzf php-5.6.16.tar.gz
+  - cd php-5.6.16/ext/pcntl/
+  - phpize && ./configure && make install
```
Comment (Sanchiz) [inline .travis.yml]: Should it be added to config file?

```
$ cd modules
$ cp pcntl.so /usr/lib/php5/<long number like 20090626>/
$ sudo gedit /etc/php5/conf.d/pcntl.ini
```
http://j7mbo.blogspot.com/2013/05/install-pcntl-in-php5-on-ubuntu.html

**Comment by podarok** [comment]
@Sanchiz can be reviewed, merged

**Pull request merged**